bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -33,6 +33,7 @@ spec:
           Networking is degraded on nodes when OVN controller is not connected to OVN southbound database connection. No networking control plane updates will be applied to the node.
       expr: |
         max_over_time(ovn_controller_southbound_database_connected[5m]) == 0
+      for: 5m
```

Comment:
Please consider making this 15 or 20 minutes, same as your other alerts.  We are experiencing severe CI issues due to this new alert.